0_fund_processing.R

```diff
@@ -1,5 +1,4 @@
-library(tidyverse)
```

Comment:
@Clare2D this file is not needed for anything, right?

Reply:
If so, maybe flag this in #144